Integrations/src/test/java/io/deephaven/integrations/learn/ComputerTest.java

```diff
@@ -23,93 +22,99 @@ public static void createTable() {
                         new double[]{5.1, 2.8, 5.7, 2.4, 7.5, 2.2, 6.4, 2.1, 7.8}
                 });
-        inputs = new Input[]{new Input(new String[]{"Column1","Column2"}, args -> args), new Input("Column3", args -> args)};
     }
 
-    static Input[] createInputs(Function<Object[], Object> gatherFunc) {
-        return new Input[]{new Input(new String[]{"Column1","Column2"}, gatherFunc), new Input("Column3", gatherFunc)};
+    private static Input[] createInputs(Function<Object[], Object>... gatherFuncs) {
+        return new Input[]{new Input(new String[]{"Column1","Column2"}, gatherFuncs[0]), new Input("Column3", gatherFuncs[1])};
     }
 
-    @Test
-    public void computeMethodTest() {
-
-        final Function<Object[], Object> func = args -> args;
-        final int batchSize = 7;
-
-        Computer computer = new Computer(table, func, inputs, batchSize);
-
-        for (int i = 0 ; i < 9 ; i++) {
-            computer.compute(i);
-        }
-
-        computer.clear();
+    private static Input[] createInputs(Function<Object[], Object> gatherFunc) {
+        return createInputs(gatherFunc, gatherFunc);
     }
 
     @Test(expected = io.deephaven.base.verify.RequirementFailure.class)
     public void nullTableTest() {
 
-        final Function<Object[], Object> func = args -> args;
+        final Input[] inputs = createInputs(args -> args);
         final int batchSize = 7;
 
-        Computer computer = new Computer(null, func, inputs, batchSize);
+        Function<Object[], Object> modelFunc = (params) ->
+        {
+            return 1;
+        };
+
+        Computer computer = new Computer(null, modelFunc, inputs, batchSize);
     }
 
     @Test(expected = io.deephaven.base.verify.RequirementFailure.class)
     public void nullFunctionTest() {
 
-        final Function<Object[], Object> func = null;
+        final Input[] inputs = createInputs(args -> args);
         final int batchSize = 7;
 
-        Computer computer = new Computer(table, func, inputs, batchSize);
+        Function<Object[], Object> modelFunc = null;
+
+        Computer computer = new Computer(table, modelFunc, inputs, batchSize);
     }
 
     @Test(expected = io.deephaven.base.verify.RequirementFailure.class)
     public void invalidBatchSizeTest() {
 
-        final Function<Object[], Object> func = args -> args;
+        final Input[] inputs = createInputs(args -> args);
         final int batchSize = 0;
 
-        Computer computer = new Computer(table, func, inputs, batchSize);
+        Function<Object[], Object> modelFunc = (params) ->
+        {
+            return 1;
+        };
+
+        Computer computer = new Computer(table, modelFunc, inputs, batchSize);
     }
 
     @Test
     public void computeGatherGetTest() {
 
-        final int batchSize = 7;
 
+        final Input[] inputs = createInputs(args -> args);
+
+        // initialize necessary reference objects and batchSize parameter
```

Comment:
Does teh comment add anything?  A better comment would explain what is going on from a conceptual level, rather than repeating what is obvious from the code.